src/App.js

```diff
@@ -14,11 +15,35 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    buttonVisibile: false,
+  };
+
+  start = () => {
+    this.setState(state => ({
+      buttonVisibile: !state.buttonVisibile,
+    }));
+  }
+
+  render() {
+    const { buttonVisibile } = this.state;
+
+    return (
+      <div className="App">
+        <h1>Goods</h1>
+        <button
+          type="button"
+          className="btn"
+          onClick={this.start}
+        >
+          {buttonVisibile ? 'Close' : 'Start'}
```

Comment:
use conditional rendering for that , it's better not render button at all than render it fist ant hide with css after